### PR TITLE
Support scope-down statements

### PR DESCRIPTION
**Pull request by Igarfinkle**
[Pivotal story](https://www.pivotaltracker.com/n/projects/2379411/stories/179148324)

Adds support for scope-down statements in some rules

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -10,48 +10,57 @@ variable "scope" {
...
   type = list(object({
-    name            = string
-    priority        = number
-    override_action = string
-    excluded_rules  = list(string)
+    name                 = string
+    priority             = number
+    override_action      = string
+    excluded_rules       = list(string)
+    scope_down_statement = statement
```
Comment (Igarfinkle) [inline variables.tf:17]: This type definition is definitely wrong. But I'm not sure how to correct it? It seems like [Statement](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/wafv2_rule_group#statement) is recursively defined.

Comment (cblkwell) [reply variables.tf:17]: Statement isn't a valid type; considering `excluded_rules` is a `list(string)` type and the structure should be similar, I suspect what you want is a `list(string)`

**Comment by dynamike** [comment]
@Igarfinkle Can this be closed?

**Comment by avanti-joshi** [comment]
confirmed with @Igarfinkle that this PR can be closed since the fix would require a fair bit of restructuring

**Pull request closed without merging**